application/common/helpers/Utils.php

```diff
@@ -206,7 +206,7 @@ public static function getFrontendConfig()
 
         $config = [];
 
-        $config['idpName'] = $params['idpName'];
+        $config['idpName'] = $params['idpDisplayName'];
```

Comment:
Not in this case, the $config array is returned as the body for the /config API call, so changing the key here changes the API.